go/lint-project.sh

```diff
@@ -227,17 +227,40 @@ if [[ "$OS_NAME" != "windows" ]]; then
     else
         # Download golangci-lint
         wget -q -O - -q https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s "$golangci_version"
+        echo "STARTING golangci-lint checks"
+        ./bin/golangci-lint version
 
         # Create a temporary filepath for the config file
         configFilepath=$(mktemp -d)"/config.yml"
         cat <<EOF > "$configFilepath"
+run:
+  timeout: 5m
+  tests: false
+  go: "$GO_VERSION"
+  skip-dirs:
+    - "cmd/*"
+    - "admin"
+    - "client"
+
+linters:
+  disable-all: true
```

Comment:
do we want to disable **all** except forbidigo?

Reply:
This PR runs golangci-lint in two passes now. The first pass only runs forbidigo and the second runs all of our other linters. It was the only way I could think of to run some linters on main code and some linters on all code.

Reply:
I see it now. Thanks for the clarification!